
Header header
float64[] joint_angles
float64[] joint_velocities
float64[] actual_torques
float64[] desired_angles
